content/Recipes/Studio-ContentAndMedia.recipe.json

```diff
@@ -3775,52 +3806,64 @@
-                    "Author": "[js: parameters('AdminUsername')]",
+                    "ModifiedUtc": "2022-09-27T14:48:54.8382078Z",
+                    "PublishedUtc": "2022-09-27T14:48:54.8576458Z",
+                    "CreatedUtc": "2022-09-27T14:48:54.8382078Z",
+                    "Owner": "4er5wqvgngp4qwjrd312gpfrnw",
+                    "Author": "etch-play",
                     "Footer": {
-                        "Logo": {
-                            "Paths": ["Logos/logo-play@1x.svg"]
+                        "ContentWidth": {
+                            "Text": "default"
+                        }
+                    },
+                    "BackgroundPart": {
+                        "BackgroundColour": {
+                            "Value": "transparent"
                         },
-                        "LogoSmall": {
-                            "Paths": ["Logos/logo-play-short@1x.svg"]
+                        "BackgroundPattern": {
+                            "Text": null
                         },
-                        "Menu": {
-                            "ContentItemIds": ["4enkkc6t2b7hvrw0swx4n5bdw9"]
+                        "InvertTextColour": {
+                            "Value": false
                         },
-                        "LogoAlternateText": {
-                            "Text": "Company logo."
+                        "BackgroundFixed": {
+                            "Value": false
+                        },
+                        "BackgroundFill": {
+                            "Value": false
                         }
                     },
                     "Content": {
                         "Widgets": [
                             {
-                                "ContentItemId": "4m4p7zmkdpk31xmr3akk9s8zd8",
+                                "ContentItemId": "41bfjhs3w075j43h361hhvcxm1",
                                 "ContentItemVersionId": null,
                                 "ContentType": "Html",
                                 "DisplayText": null,
                                 "Latest": false,
                                 "Published": false,
-                                "ModifiedUtc": "2020-08-04T22:08:34.4741416Z",
+                                "ModifiedUtc": "2022-09-27T14:48:54.7937284Z",
                                 "PublishedUtc": null,
                                 "CreatedUtc": null,
                                 "Owner": null,
-                                "Author": "[js: parameters('AdminUsername')]",
+                                "Author": "etch-play",
                                 "Html": {
                                     "Content": {
-                                        "Value": "<p>© 2020 Etch Play. <a href=\"#\">Privacy policy</a></p>"
+                                        "Value": "<p>© 2020 Etch Play. <a href=\\\"#\\\">Privacy policy</a></p>"

```

Comment:
Can we make this 2022, or better yet, a dynamic year? Just a suggestion to consider, but I'll approve regardless :)

Reply:
Plan is to replace with the new "Liquid" widget that will be coming once we've completed the 1.4 upgrade (https://github.com/EtchUK/Etch.OrchardCore.Widgets/pull/111). This will allow us to make it dynamic.